php-nginx/8.5/Dockerfile

```
@@ -0,0 +1,119 @@
+
+COPY docker-entrypoint.sh /
+COPY 10-listen-on-ipv6-by-default.sh /docker-entrypoint.d
+COPY 15-local-resolvers.envsh /docker-entrypoint.d
+COPY 20-envsubst-on-templates.sh /docker-entrypoint.d
+COPY 30-tune-worker-processes.sh /docker-entrypoint.d
+ENTRYPOINT ["/docker-entrypoint.sh"]
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+WORKDIR /app
+EXPOSE "80"
```

Hadolint (2.14.0)

[error] unexpected '"'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)